MPEdn/MPEdnDateCodec.m

```diff
@@ -29,13 +30,17 @@ + (void) initialize
     // NSDateFormatter is *very* slow to create, pre-allocate one
     // NB NSDateFormatter is thread safe only in iOS 7+ and OS X 10.9+
     // TODO warn if being compiled on a platform where this is unsafe
-    dateFormatter = [NSDateFormatter new];
+    dateFormatterRead = [NSDateFormatter new];
+    dateFormatterWrite = [NSDateFormatter new];
+
+    dateFormatterRead.locale = [NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"];
```

Comment:
This is recommended by Apple:

https://developer.apple.com/documentation/foundation/nsdateformatter#2528261

https://developer.apple.com/library/archive/qa/qa1480/_index.html